src/main/resources/data.sql

```diff
@@ -0,0 +1 @@
+INSERT INTO televisions (name, brand, price, wifi, voice_control) VALUES ('TV123', 'Samsung', 400, true, false);
```

Comment:
Handig zo'n data.sql om te zorgen dat je altijd data in je database hebt staan. Je kunt hier misschien nog wat meer data toevoegen, zoals ook de sales informatie om te zien wat er gebeurt als je de bonus opdracht gaat implementeren